Tests/ToolBox/LeanDataWriterTests.cs

```diff
@@ -171,5 +178,193 @@ public void LeanDataWriter_CanWriteCrypto()
 
             Assert.AreEqual(data.First().Value.Count(), 3);
         }
+
+        [Explicit("Data copying and reading test")]
```

Comment:
By using `Explicit` these test aren't being run by the CI, think we should remove it? `CanDownloadAndSave(Equity,Quote,Hour)` is throwing some exceptions, my guess is because the file doesn't exist?

Reply:
Gotcha 👌🏽 Removed explicit tag.

After reviewing `CanDownloadAndSave(Equity,Quote,Hour)`, yes you are correct, it was reading in TradeBars from our Hourly folder as QuoteBars.